### DISCUSSION TITLE
Remove Curse of Binding armor if player has weakness effect

### DISCUSSION THREAD

**Discussion by hirata-estate** [PurpurMC/Purpur#557] [__Completed__] [answered]
Allow the player to remove their curse of binding armor when they have a weakness effect/potion applied to them.

**Comment by svdgoor** [comment]
Should be an option, not on by default imo

**Reply by granny** [reply]
Nothing in purpur is enabled by default, this request is no different

**Reply by svdgoor** [reply]
k soz

**Comment by encode42** [comment, marked as answer]
Closed in https://github.com/pl3xgaming/Purpur/commit/947158fd5fbd4d0dfcda02654aef0f849b151946
